FPSCamera/Cam/FollowCam.cs

```diff
@@ -93,6 +110,44 @@ protected virtual bool _SwitchTarget(IDType newID)
             return true;
         }
 
+        public Position GetSmoothLookPos()
+        {
+            uint targetFrame = _target.GetTargetFrame();
+            Position pos1 = _GetFrame(targetFrame - 2 * 16U);
+            Position pos2 = _GetFrame(targetFrame - 0 * 16U);
+            float t = ((targetFrame & 15U) + SimulationManager.instance.m_referenceTimer) * 0.0625f;
```

Comment:
if you wondering where do I get all these weird numbers from its copy pasted from decompiler.